pkg/registry/core/pod/storage/eviction.go

```diff
@@ -297,7 +275,11 @@ func setPreconditionsResourceVersion(deleteOptions *metav1.DeleteOptions, resour
 // without checking PDBs.
 func canIgnorePDB(pod *api.Pod) bool {
```

Comment:
called from https://github.com/kubernetes/kubernetes/blob/master/pkg/registry/core/pod/storage/eviction.go#L158-L180 which adds the precondition we want.